hugo/layouts/partials/header.html

```diff
@@ -54,7 +54,7 @@
                         <a class="btn btn-sm btn-highlight-light" href="https://go.sourced.tech/engine">Request a Demo</a>
                     </li>
                     <li class="nav-item nav-btn">
-                        <a class="btn btn-sm btn-highlight" href="https://docs.sourced.tech/engine/#quickstart">Install Engine</a>
+                        <a class="btn btn-sm btn-highlight" href="https://go.sourced.tech/engine-download">Install source{d} Engine</a>
```

Comment:
Same as above review comment. The context around it makes for a very strong and obvious connection the engine.

Reply:
Thanks @marnovo ! Done.